client/packages/common/src/intl/locales/en/common.json

```diff
@@ -20,7 +20,7 @@
   "button.save": "Save",
   "button.select-a-color": "Select a color",
   "button.try-again": "Try again",
-  "description.already-issued": "Quantity already issued.",
+  "description.unissued": "Unissued amount.",
```

Comment:
Thought this was more appropriate since the total decreases once an outbound shipment has been created